db/schema.rb

```diff
@@ -1202,6 +1202,48 @@
   add_index "weams", ["institution"], name: "index_weams_on_institution", using: :btree
   add_index "weams", ["state"], name: "index_weams_on_state", using: :btree
 
+  create_table "yellow_ribbons", force: :cascade do |t|
+    t.string   "facility_code"
+    t.string   "school_name_in_yr_database"
+    t.string   "school_name_in_weams"
+    t.string   "campus"
+    t.string   "street_address"
+    t.string   "city"
+    t.string   "state"
+    t.string   "zip"
+    t.string   "public_private"
+    t.string   "degree_level"
+    t.string   "division_professional_school"
+    t.string   "contribution_amount"
+    t.string   "updated_for_2011_2012"
+    t.string   "missed_deadline"
+    t.string   "ineligible"
```

Comment:
Same desire for a boolean here, but more of the "- Please make a selection -" values.